src/test/java/next/reflection/Junit3TestRunner.java

```diff
@@ -2,10 +2,16 @@
 
 import org.junit.jupiter.api.Test;
 
+import java.lang.reflect.Method;
+
 public class Junit3TestRunner {
     @Test
     public void run() throws Exception {
         Class<Junit3Test> clazz = Junit3Test.class;
-        // TODO Junit3Test에서 test로 시작하는 메소드 실행
+        for (Method method : clazz.getDeclaredMethods()) {
+            if (method.getName().startsWith("test")) {
+                method.invoke(clazz.newInstance(), null);
```

Comment:
```suggestion
                method.invoke(clazz.getDeclaredConstructor().newInstance(), null);
```

해당 메서드는 Deprecated 되었네요 ㅎ

https://stackoverflow.com/questions/53257073/java-9-replace-class-newinstance